api/src/usersAvatar/controllers/usersAvatar-controller.ts

```diff
@@ -14,13 +15,26 @@ import {
     ApiTags,
     ApiUnauthorizedResponse
 } from "@nestjs/swagger";
+import { createReadStream } from "fs";
 
 @ApiBearerAuth("defaultBearerAuth")
 @ApiTags("users-avatar")
 @Controller("users-avatar")
 class UsersAvatarController {
     constructor(private readonly usersAvatarService: UsersAvatarService) {}
 
+    @ApiUnauthorizedResponse()
+    @ApiOkResponse()
+    @ApiNotFoundResponse()
+    @Get(":id")
+    async getFile(@Param("id") id: number): Promise<StreamableFile> {
+        const avatar = await this.usersAvatarService.findOne(id);
+
+        const file = createReadStream(process.cwd() + "\\" + avatar.sourcePath);
```

Comment:
For me, it should be in usersAvatarService as separated method but up to You as well.

Reply:
ye, that's right i will refactor it.

Reply:
Done